ecosystem/README.md

```diff
@@ -146,6 +147,10 @@ From there, the following process will happen:
   * If the SEP requires active maintenance, such as having an open schema, it should be marked at
     `Active`.
   * Otherwise, the SEP should be marked as `Final`.
+  
+### Regression
+* It is possible for a SEP to move from `Active` to `Draft` or `Deprecated` if it is never adopted, or is abandoned by the community.
```

Comment:
A `Draft` proposal should probably move to `Rejected` rather than `Deprecated`. Otherwise there's not much distinction between the two?

Reply:
Oh, I misread the sentence. It's saying that `Active` can move to `Draft`/`Deprecated`, not that `Active`/`Draft` can move to `Deprecated`. Sorry!

Reply:
Yeah i'm not so sold on this sentence. I want to be able to move back to `draft` because there are some seps that seemingly would be useful in the future so i don't want to put them in a rejected status

Reply:
Maybe break it up into two statements.
```suggestion
* It is possible for a SEP to move from `Active` to `Draft` if it is not adopted and it needs further work.
* It is possible for a SEP to move from `Active` to `Deprecated` if it is never adopted and abandoned.
```